### DISCUSSION TITLE
Few questions/ideas

### DISCUSSION THREAD

**Discussion by Idobenhamo** [Feel-ix-343/markdown-oxide#66] [Ideas]
Wanted to ask about few features that could help a lot:
- Any chance for adding tags in the metadata like the aliases?
- Maybe add in the config a folder location for the daily notes?
- Adding auto complete for wiki links for images like obsidian?

Thanks again for your awesome work!

**Comment by Feel-ix-343** [comment]
Hey! 

- Folder location is being tracked here #56 
- I just created an issue #67 for the tags. 
- I just created #68  for non-mdfile completions

Thanks for the comment!

**Reply by Idobenhamo** [reply]
Hi,
I hope the code refactoring is going well, just wanted to ask a tiny question.
Each file of mine is starting with level 1 heading of the file name. The metadata (YAML) should be bellow the first header or above it?
I am having a bit of trouble to make the aliases work... The aliases can be placed inline or just as indented list?

Thank you for everything! @Feel-ix-343

**Reply by Feel-ix-343** [reply]
Thank you!!!

Right now the code assumes that the aliases will start on the first line of the file, so headers should go below.

I have not researched if this is markdown spec, but it is very typical

**Reply by Feel-ix-343** [reply]
And the aliases can either be an indented list or a list like `aliases: [one name, two name]` or `aliases: ["one name", "two name"]`

**Reply by Idobenhamo** [reply]
Thank you!
May I ask how the progress of refactoring is going? I hope I would have some free time soon and I really want to contribute.

BTW, the readme file is pretty long & heavy right now. have you think about separating it to a few files? to make kind of a wiki...

**Reply by Feel-ix-343** [reply]
Hey! Thanks for checking in!

Currently, I am in the midst of exams at school, and I do not have much time to write new code. My situation will change in the next 2 weeks, though. 

I agree about the README. I think a wiki would be a great start; my true vision is a beautiful website showing off all of markdown-oxide's features and highlighting its compatibility with one's favorite text editor. 

I'll open an issue for this for now.

**Reply by Idobenhamo** [reply]
There is a way to open the daily note file in helix?

**Reply by Feel-ix-343** [reply]
Hmm I am not sure. I assume helix does not support lsp commands but Ill check.

**Reply by Feel-ix-343** [reply]
Oh interesting; it seems that commands are supported. Ill open an issue for this: it will involve a helix PR.

**Reply by Feel-ix-343** [reply]
#115

**Reply by Idobenhamo** [reply]
King thank you for everything!

**Reply by Idobenhamo** [reply]
@Feel-ix-343 I think the tags have an issue with tags that start with number. For example: #3D_printed

**Reply by Feel-ix-343** [reply]
Hey!

This will be fixed in v1 with the new tree-sitter based parser.
